experimental/Schemes/IdealSheaves.jl

```diff
@@ -268,7 +269,7 @@ function simplify!(I::IdealSheaf)
   return I
 end
 
-@doc raw"""
+@Markdown.doc """
```

Comment:
```suggestion
@doc  raw"""
```
Why did you change it? Does @doc raw""" not work for you?